### ISSUE TITLE
Lists of Categored websites

### ISSUE THREAD

**Issue by CyberCrime-Stoppers** [CyberCrime-Stoppers/Lists-of-website#1]
___________________________
Search Engine
___________________________
ask
baidu
bing
dogpile
duckduckgo
ecosia
excite
gibiru
gigablast
google
mojeek
proton
qwant
search-encrypt
search.encrypt
startpage
swisscows
webcrawler
wolframalpha
yahoo
yandex
yelp

___________________________
Canada Provance
___________________________
alberta
british
brunswick
columbia
edward
island
labrador
manitoba
newfoundland
northwest
nova
nunavut
ontario
quebec
saskatchewan
territories
yukon

___________________________
Countires
___________________________
afghanistan
african
albania
algeria
andorra
angola
antigua
argentina
armenia
australia
austria
azerbaijan
bahamas
bahrain
bangladesh
barbados
belarus
belgium
belize
benin
bhutan
bolivia
bosnia
botswana
brazil
brunei
bulgaria
burkina-faso
burundi
cabo verde
cambodia
cameroon
canada
chad
chile
china
colombia
comoros
congo
cote
croatia
cuba
cyprus
czech
denmark
djibouti
dominica
dominican
east-timor
east.timor
easttimor
ecuador
egypt
el-salvador
equatorial
eritrea
estonia
eswatini
ethiopia
fiji
finland
france
gabon
gambia
georgia
germany
ghana
greece
grenada
guatemala
guinea
guinea-bissau
guyana
haiti
honduras
hungary
iceland
india
indonesia
iran
iraq
ireland
israel
italy
jamaica
japan
jordan
kazakhstan
kenya
kiribati
korea
kosovo
kuwait
kyrgyzstan
laos
latvia
lebanon
lesotho
liberia
libya
liechtenstein
lithuania
luxembourg
madagascar
malawi
malaysia
maldives
mali
malta
marshall
mauritania
mauritius
mexico
micronesia
moldova
monaco
mongolia
montenegro
morocco
mozambique
myanmar
namibia
nauru
nepal
netherlands
new-zealand
nicaragua
niger
nigeria
north-macedonia
norway
oman
pakistan
palau
palestine
panama
papua
paraguay
peru
philippines
poland
portugal
qatar
rica
romania
russia
rwanda
saint
saint-lucia
saint-vincent
samoa
san-marino
sao
saudi-arabia
senegal
serbia
seychelles
sierra
singapore
slovakia
solomon
somalia
spain
sri-lanka
sudan
suriname
sweden
switzerland
syria
taiwan
tajikistan
tanzania
thailand
togo
tonga
trinidad
tunisia
turkey
turkmenistan
tuvalu
uganda
ukraine
united-arab-emirates
united-kingdom
united-states
uruguay
uzbekistan
vanuatu
vatican
venezuela
vietnam
yemen
zambia
zimbabwe
___________________________
Tech Ginants
___________________________
amd
apple
asus
azure
cable
cisco
console
dell
huawei
ibm
intel
lenovo
meta
microsoft
nvidia
oracle
rogers
samsung
shaw
shawplus
sony
tesla
xbox
xiaomi

_____________________________
Developer & Programming
_____________________________
codecademy
codepen
coursera
freecodecamp
geeksforgeeks
github
hackerrank
leetcode
mdn
stackoverflow
udemy
w3schools
_____________________________
Open-Source + Linux 
_____________________________
archlinux
centos
debian
dubuntu
elementary
fedora
hubuntu
kali
kali-linux
kalilinux
kubuntu
linuxmint
lubuntu
manjaro
mubuntu
opensuse
qubuntu
ubuntu
zorin
zorin-os
zorin.os
zorinos
_____________________________
Open-Source + Linux - Domain
1- UBUNTU
_____________________________
            1

ca.archive.ubuntu.com
cn.archive.ubuntu.com
in.archive.ubuntu.com
jp.archive.ubuntu.com
mx.archive.ubuntu.com
ru.archive.ubuntu.com
uk.archive.ubuntu.com
us.archive.ubuntu.com

security.ubuntu.com
tcp.ca.archive.ubuntu.com
tcp.security.ubuntu.com
_http._tcp.ca.archive.ubuntu.com
_http._tcp.security.ubuntu.com
            END 

_____________________________
Social Media
_____________________________
500px
api-facebook
behance
discord
dribbble
ello
facebook
fbcdn
fbcdn-photos
fbcdn-profile
fbcdn-sphotos
fbexternal
flickr
gab
instagram
linkedin
login-facebook
medium
meetup
mewe
minds
nextdoor
parler
pinterest
quora
reddit
ritetag
snapchat
static-ak-fbcdn
telegram
tiktok
tumblr
twitter
vero
vrchat
wechat
whatsapp

_____________________________
Online Dating
_____________________________
bumble
coffeemeetsbagel
eharmony
elitesingles
hinge
match
okcupid
plentyoffish
tinder
zoosk

_____________________________
Email Services
_____________________________
aol
apple
disroot
fastmail
gmail
gmx
icloud
kolab
lavabit
luxsci
mail
mailfence
neomailbox
openmailbox
outlook
riseup
thexyz
titanfile
vfemail
yahoo
yandex.mail
zoho
_____________________________
Paid Email(s)
_____________________________
countermail
criptext
hushmail
mailbox
posteo
protonmail
runbox
scryptmail
startmail
tutanota
zohomail
___________________________
Moding
___________________________
armaholic
civfanatics
curseforge
dbcraft
gamebanana
gta5-mods
loverslab
mod-nexus
mod.nexus
moddb
moddingway
modnexus
modthesims
nexus-mods
nexus.mods
nexusmods
nfsmods
shadderpack
shadders
shadderspack
simtropolis
steam-workshop
steam.workshop
steamworkshop
___________________________
Video Games
___________________________
agar.io
angry-birds
angry.birds
angrybirds
blockheads
blockland
business-tycoon-online
business.tycoon.online
businesstycoononline
call-of-duty
call-of-duty-warzone
call.of.duty
call.of.duty.warzone
callofduty
callofdutywarzone
cookie-clicker
cookie.clicker
cookieclicker
crazygames
creativerse
doom
epicmathgames
fortnite
freemathgames
geoguessr
gmod
grandtheftauto
happywheels
kerbal-space-program
kerbal.space.program
kerbalspaceprogram
kogmama
marketglory
minecraft
opensandbox
playground
powerline.io
rgpmaker
roblox
rollercoaster
run-3
run.3
run3
sandbox
sinespace
slope
sonicboom
stickgames
theblockheads
trove
warcraft
wizard101
___________________________
Game Launcher
___________________________
battle
cyberpunk-2077
cyberpunk.2077
cyberpunk2077
discord
ea-games
ea.com
epicgames
gamefly
gamejolt
gamestop
gog
help.ea.com
humble
ir.ea.com
itch
kongregate
minecraft
newgrounds
nintendo
nvidia
origin
parsec
playstation
razer
roblox
rockstargames
stadia
steam
steampowered
unisoft
xbox
___________________________
WebBrowsers
___________________________
basilisk
brave
browser-avant
browser-beaker
browser-dissenter
browser-epicprivacy
browser-ghost
browser-min
browser-puffin
browser-tenta
browser-whiskermenu
browser-yandex
chrome-google
cliqs
coc-coc
colibri
comodo
comodo-dragon
dooble
dooscape
edge
falkon
firefox
gnu-icecat
gnuzilla
greenbrowser
internet-explorer
konqueror
maxthon
midori
mosaic
mozilla
netscape-navigator
opera
otter-browser
palemoon
polypane
puffin
qtweb
safari
seamonkey
slimjet
srware
srware-iron
superbird
tor-browser
tor.browser
torbrowser
torch-browser
torproject
uc-browser
vivaldi
waterfox

___________________________
Cloud Stoage Services
___________________________
amazon.drive
amazondrive
box
drive.amazon
dropbox
gdpr
icloud
mediafire
mega.io
onedrive
pcloud
slither.io
spideroak
sugarsync
sync
zoho-docs
zoho.docs
zohodocs
___________________________
Educational
___________________________
alison
brainpop
futurelearn
harvard
khan
mit
natinalgeographic
pbs
quizlet
stanford
ted-ed
teded
ubacity
___________________________
Badware
___________________________
ad
ad.lionmobi
adcaffe
alt2-mtalk
alt4-mtalk
alt5-mtalk
anythinktech
bingviz
bloody-disgusting
canadagold
chatplus
claimit
claimitpagedev
cloudfront
comedy-radio
crashlyticsreports
dell
dl197
dl230
dl231
dldellcom
drive2
dzen
e-mail
edge-mqtt
filemate10
filemate17
flipboard
freemoney
gnst
goggle
gogle
iphoneitalia
itdratinia
izatcloud
link
magiceden
mail
minigames
mmminigames
mobileconfiguration
mobsms
mobsms.guard
msk
my-mail
newmode
octavius
onezapp
online
openai
ossshucklean
phonejapan
razlozhi
refitdratiniacom
rimage
roku
rul
science-output
silent
smartarget
smartsuppcdn
smokingmeatforums
speech
telemetry
theblogrill
thehabitatstore
vibeaconstr
vk
xerxes
xerxessecure
xtracloud
yun-hl
___________________________
DNS Providers - Domain Name Servers
___________________________
cloudflare
dnsgoogle
nextdns
opendns
quad9

_____________________________
Pornography
_____________________________
123sex
18gayboys
18teen
18teenpornpics
18teenpussyfuck
18video
999live
applive
automarket
biphim
boy18tube
boymoviedome
boyngon
callboy427
catewalk
caysex
chich
chichlive
chichs
chichslive
chick
chicklove
chicks
chim18
cliphot
cliphotvn
clipnong
clipsexvn
dam18
dam3x
dam69
dithau
freesex
girlkiemdinh
girlsporn
gockhaut
hay3x
hotgirls
mbbg
megaideb
messagex
moonlive
msexhayvc
msexsub
pornhub
sex
xnxx
xvideo
xvideo1
xvideos
xvideos1
xvideos102
xvideos21
xvideos888
xvideos98
xviet
xxnx
xxx
xxxdam
xxxner

_____________________________
Job Search
_____________________________
51job
angellist
bayt.com
careerbuilder
craigslist
dice
flexjobs
glassdoor
indeed
indeed-canada
indeed.canada
jobberman
linkedin
monster
naukri.com
seek
simplyhired
snagajob
the-muse
the.muse
usajobs
ziprecruiter

_____________________________
News
_____________________________
abcnews
al-jazeera
al.jazeera
aljazeera
associatedpress
bbc
bloomberg
cbs
cbsc
cnbc
cnn
foxnews
huffpost
nbc
newyork
newyorktimes
npr
politico
reuters
theguardian
thewallstreetjournal
thewashingtonpost
usatoday
vox
_____________________________
Online Shopping
_____________________________
alibaba
aliexpress
amazon
asos
bestbuy
ebay
etsy
flipkart
jd.com
lazada
mercadolibre
newegg
overstock
rakuten
shopify
taobao
target
walmart
wish
zalando

_____________________________
Antivirus & Database + Credentials
+ Browser/Computer Extensions
_____________________________
avast
avg
avira
eset-nod32
eset.nod32
esetnod32
kaspersky
malwarebytes
mcafee
sophos
trend-micro
trend.micro
trendmicro

_____________________________
VPN(s)
_____________________________
cyberghost
expressvpn
ipvanish
nordvpn
nortonvpn
privateinternetaccess
surfshark
tunnelbear
vyprvpn
windscribe

_____________________________
Password Manager
_____________________________
1password
adblock
adblock-plus
adblock.plus
adblocker
adblockplus
adguard
bitwarden
blur
dashlane
disconnect
ghostery
https-everywhere
keeper
lastpass
privacy-badger

_____________________________
News & Content
_____________________________
feedly
feedly-mini
feedly.mini
feedlymini
instapaper
mercurty-reader
mercury
mercury.reader
mercuryreader
tldr
_____________________________
Developer Tools
_____________________________
editthiscookie
jsonview
octotree
user-agent
useragent
vue.js
wappalyzer
web-developer
webdeveloper
_____________________________
Entertainment & Customization:
_____________________________
darkreader
google-translate
google.translate
googletranslate
imagus
stylus
tampermonkey

_____________________________
All Blogging Websites
_____________________________
anchor
blogger
drupal
ghost
hubspot
joomla
livejournal
medium
penzu
postach.io
squarespace
substack
svbtle
tumblr
typepad
weebly
wix
wordpress
wordpress.org
write.as

_____________________________
Video and Music Streaming Platform
_____________________________
apple-music
apple-tv
apple.music
apple.tv
bitchute
brightcove
crunchyroll
dailymotion
deezer
disneyplus
espn
funimation
hbo
hulu
jw-player
jw.player
jwplayer
loom
metacafe
netflix
peertube
rumble
soundcloud
spotify
tidal
twitch
veoh
vevo
vimeo
wistia
youtube

____________________________
Routers+Modems, Corp. Domain
1- TP-LINK
____________________________
            1

n-deventry-gw.tplinkcloud.com 
 
           END
_____________________________
Streaming Domain
1- Netflix
2- Vimeo
3-
_____________________________
            1

logs.netflix.com
preapp.prod.partner.netflix.net
push.prod.netflix.com
            
            END
            ---
             2
 
www.vimeo.com

_____________________________
Netflix Domain
_____________________________
api-global.netflix.com

_____________________________
techgiant_pings
_____________________________
edge.microsoft.com
go.microsoft.com
ipv6.msftconnecttest.com
msftconnecttest.com
www.msftconnecttest.com
_____________________________
IP Look Up
_____________________________
www.myip.com
whatismyip.com
whatismyipaddress.com

___________________________
Domain Search Engine
___________________________
accounts.google.com
play.google.com
tools.google.com

ade.googlesyndication.com
pagead2.googlesyndication.com

www3.l.google.com
photos-ugc.l.googleusercontent.com
dai.google.com
dl.google.com
dns.google.com
yt3.ggpht.com
___________________________
Google SafeSearch
___________________________
forcesafesearch.google.com

___________________________
Google Countires
___________________________
google.at
google.be
google.bg
google.ch
google.cl
google.cn
google.co.cr
google.co.id
google.co.il
google.co.in
google.co.jp
google.co.ke
google.co.kr
google.co.nz
google.co.th
google.co.uk
google.co.za
google.com.au
google.com.br
google.com.do
google.com.eg
google.com.gr
google.com.hk
google.com.mx
google.com.my
google.com.ng
google.com.pe
google.com.ph
google.com.pk
google.com.sa
google.com.sg
google.com.tr
google.com.tw
google.com.ua
google.com.uk
google.com.uy
google.com.ve
google.com.vn
google.cz
google.de
google.dk
google.ee
google.es
google.fi
google.fr
google.hu
google.ie
google.it
google.nl
google.no
google.pl
google.pt
google.ro
google.ru
google.se
google.sk
___________________________
Google Android
___________________________
android.clients.google.com

___________________________
gstatic
___________________________
connectivitycheck.gstatic.com
uhkl8q-dnsotls-ds.metric.gstatic.com
rkpxu1-dnsotls-ds.metric.gstatic.com
mljpy8-dnsotls-ds.metric.gstatic.com
qmhwn3-dnsotls-ds.metric.gstatic.com
5idtaz-dnsotls-ds.metric.gstatic.com
b4rqhj-dnsotls-ds.metric.gstatic.com
qj08ni-dnsotls-ds.metric.gstatic.com
kpmt6-dnsotls-ds.metric.gstatic.com
fthnl-dnsotls-ds.metric.gstatic.com
bhfqf0-dnsotls-ds.metric.gstatic.com
zx2tb4-dnsotls-ds.metric.gstatic.com
n0ewyu-dnsotls-ds.metric.gstatic.com
d2xknj-dnsotls-ds.metric.gstatic.com
irzaop-dnsotls-ds.metric.gstatic.com
zj7v8d-dnsotls-ds.metric.gstatic.com
spiipv-dnsotls-ds.metric.gstatic.com
cy1p0s-dnsotls-ds.metric.gstatic.com
jei6ad-dnsotls-ds.metric.gstatic.com
oaWWHA-dnsotls-ds.metric.gstatic.com
2Dpu5S-dnsotls-ds.metric.gstatic.com
J3pom3-dnsotls-ds.metric.gstatic.com
NbkPWW-dnsotls-ds.metric.gstatic.com
k0qhB0-dnsotls-ds.metric.gstatic.com
Wr5re-dnsotls-ds.metric.gstatic.com
FNleTd-dnsotls-ds.metric.gstatic.com
jqQeA-dnsotls-ds.metric.gstatic.com
h5jCEu-dnsotls-ds.metric.gstatic.com

_____________________________
Google API(s)
_____________________________
deviceintegritytokens-pa.googleapis.com
embeddedassistant.googleapis.com
fonts.googleapis.com
geller-pa.googleapis.com
mobileconfiguration-pa.googleapis.com
play.googleapis.com
storage.googleapis.com
telephonyspamprotect-pa.googleapis.com
youtube.googleapis.com
_____________________________
Google Dot .LAN
_____________________________
alt8-mtalk.google.com.lan
connectivitycheck.gstatic.com.lan
www.google.com.lan
_____________________________
Google UserContent
_____________________________
play-lh.googleusercontent.com

_____________________________
GOOGLE VIDEO DOMAIN
_____________________________
r1sngvbxgntt1e7googlevideocom
r1sngvbxgntt1edgooglevideocom
r2sngvbxgntt1edgooglevideocom
r3sngvbxgntt1e6googlevideocom
r4sngvbxgntt1ezgooglevideocom
r5sngvbxgntt1e6googlevideocom
r5sngvbxgntt1eegooglevideocom

rr1---sn-gvbxgn-tm0l.googlevideo.com
rr1---sn-gvbxgn-tt1e7.googlevideo.com
rr1---sn-gvbxgn-tt1ee.googlevideo.com
rr1---sn-tt1e7nls.googlevideo.com
rr1---sn-tt1elnel.googlevideo.com
rr2---sn-gvbxgn-tm0l.googlevideo.com
rr2---sn-gvbxgn-tt1ed.googlevideo.com
rr2---sn-tt1e7nlz.googlevideo.com
rr2---sn-tt1elnel.googlevideo.com
rr2---sn-vgqsknsk.googlevideo.com
rr2---sn-vgqsrnzz.googlevideo.com
rr3---sn-gvbxgn-tt1ed.googlevideo.com
rr3---sn-gvbxgn-tt1es.googlevideo.com
rr3---sn-tt1elnel.googlevideo.com
rr3---sn-vgqsrnsd.googlevideo.com
rr4---sn-gvbxgn-tm0l.googlevideo.com
rr4---sn-gvbxgn-tt1ee.googlevideo.com
rr4---sn-gvbxgn-tt1el.googlevideo.com
rr4---sn-gvbxgn-tt1ez.googlevideo.com
rr4---sn-tt1e7nlz.googlevideo.com
rr5---sn-gvbxgn-tm0l.googlevideo.com
rr5---sn-tt1elnel.googlevideo.com
rr5---sn-vgqsrnll.googlevideo.com
rr6---sn-gvbxgn-tm0l.googlevideo.com
rr7---sn-gvbxgn-tm0l.googlevideo.com
rr7---sn-gvbxgn-tt1ez.googlevideo.com
rr8---sn-gvbxgn-tt1e6.googlevideo.com
rr8---sn-gvbxgn-tt1ez.googlevideo.com
rr3---sn-nx57ynsk.googlevideo.com
rr3.sn-nx57ynsk.googlevideo.com
rr1.sn-nx57ynlk.googlevideo.com
rr1---sn-nx57ynlk.googlevideo.com

rr1sngvbxgntt1edgooglevideo.com
rr2sngvbxgntt1e7googlevideo.com
rr2sngvbxgntt1elgooglevideo.com
rr2sngvbxgntt1esgooglevideo.com
rr2sngvbxgntt1ezgooglevideo.com
rr3sngvbxgntt1edgooglevideo.com
rr3sngvbxgntt1eegooglevideo.com
rr3sngvbxgntt1esgooglevideo.com
rr4sngvbxgntt1eegooglevideo.com
rr4sngvbxgntt1elgooglevideo.com
rr5sngvbxgntt1e7googlevideo.com
rr5sngvbxgntt1ezgooglevideo.com
rr8sngvbxgntt1e6googlevideo.com

___________________________
YouTube
___________________________
youtube-ui.l.google.com
yt3.ggpht.com

___________________________
Google .Mtalk.
___________________________
mtalk.google.com
alt1-mtalk.google.com
alt3-mtalk.google.com

___________________________
Google Analytics
___________________________
google-analytics.com
facvicon-www.google-analytics.com

___________________________
Google Doubleclick
___________________________
static.doubleclick.net
ad.doubleclick.net

___________________________
Manager
___________________________
googletagmanager.com
www.googletagmanager.com

___________________________
Microsoft
___________________________
8cb691e535702e64106a3948d54b901798889ee3.cws.conviva.com
account.activedirectory.windowsazure.com
accounts.accesscontrol.windows.net
adminwebservice.microsoftonline.com
api.passwordreset.microsoftonline.com
arc.msn.com
assets.msn.com
auth.microsoft.com
autologon.microsoftazuread-sso.com
becws.microsoftonline.com
ccs.login.microsoftonline.com
clientconfig.microsoftonline-p.net
collections.mp.microsoft.com
companymanager.microsoftonline.com
compliance.microsoft.com
consumer-licensing-aks2aks.md.mp.microsoft.com.akadns.net
continuum.dds.microsoft.com
defender.microsoft.com
device.login.microsoftonline.com
eus2-13.frontdoor.licensing.commerce.microsoft.com
fs.microsoft.com
graph.microsoft.com
graph.windows.net
img-prod-cms-rt-microsoft-com.akamaized.net
licensing.mp.microsoft.com
login-us.microsoftonline.com
login.microsoft.com
login.microsoftonline-p.com
login.microsoftonline.com
login.windows.net
logincert.microsoftonline.com
loginex.microsoftonline.com
lync.com
mail.protection.outlook.com
mobile.events.data.microsoft.com
msftidentity.com
msidentity.com
mx.microsoft
nexus.microsoftonline-p.com
ntp.msn.com
o516213.ingest.sentry.io
office.live.com
officeapps.live.com
online.office.com
outlook.cloud.microsoft
outlook.office.com
outlook.office365.com
passwordreset.microsoftonline.com
protection.office.com
protection.outlook.com
provisioningapi.microsoftonline.com
pti.store.microsoft.com
purview.microsoft.com
security.microsoft.com
self.events.data.microsoft.com
settings-win.data.microsoft.com
sharepoint.com
sls.update.microsoft.com
slscr.update.microsoft.com
smtp.office365.com
teams.cloud.microsoft
teams.microsoft.com
trdocentralusx1vmss.ipv6.microsoft.com.akadns.net
turing-writingassistance.edge.microsoft.com
watson.events.data.microsoft.com
www.msftconnecttest.com

___________________________
Microsoft Azure
___________________________
ns1-05.azure-dns.com
onedsblobprdwus15.westus.cloudapp.azure.com
ns1-02.azure-dns.com

___________________________
Microsoft Defender
___________________________
nav.smartscreen.microsoft.com

_____________________________
Xbox Services
_____________________________
accounts.xboxlive.com
catalog.gamepass.com
cert.mgt.xboxlive.com
clubhub.xboxlive.com
comments.xboxlive.com
device.auth.xboxlive.com
dlassets-ssl.xboxlive.com
editorial.xboxlive.com
eplists.xboxlive.com
instance.mgt.xboxlive.com
notificationinbox.xboxlive.com
notifier.xboxlive.com
ntp.ubuntu.com
peoplehub.xboxlive.com
privacy.xboxlive.com
privacy.xboxlive.comaccounts.xboxlive.com
profile.xboxlive.com
rta.xboxlive.com
sessiondirectory.xboxlive.com
settings-win.data.microsoft.comclubhub.xboxlive.com
social.xboxlive.com
title.auth.xboxlive.com
title.mgt.xboxlive.com
titlehub.xboxlive.com
titlestorage.xboxlive.com
userpresence.xboxlive.com
userstats.xboxlive.com
usertitles.xboxlive.com
www.xboxab.com
xblmessaging.xboxlive.com
xbox.ipv6.microsoft.com
xboxab.com
xflight.xboxlive.com
xgrant.xboxlive.com
xkms.xboxlive.com
xnotify.xboxlive.com
xsts.auth.xboxlive.com
xbaccessories.blob.core.windows.net

___________________________
Java Domain + Trackers
___________________________
js.stripe.com

___________________________
Unknown
___________________________
8cb691e535702e64106a3948d54b901798889ee3.cws.conviva.com

___________________________
Google Video
___________________________
2dpu5sdnsotlsdsmetricgstatic.com
5idtazdnsotlsdsmetricgstatic.com
b4rqhjdnsotlsdsmetricgstatic.com
bhfqf0dnsotlsdsmetricgstati.ccom
connectivitycheckgstatic.com
cy1p0sdnsotlsdsmetricgstatic.com
d2xknjdnsotlsdsmetricgstatic.com
fnletddnsotlsdsmetricgstatic.com
fthnldnsotlsdsmetricgstatic.com
irzaopdnsotlsdsmetricgstatic.com
j3pom3dnsotlsdsmetricgstatic.com
jei6addnsotlsdsmetricgstatic.com
jqqeadnsotlsdsmetricgstatic.com
k0qhb0dnsotlsdsmetricgstatic.com
kpmt6dnsotlsdsmetricgstatic.com
mljpy8dnsotlsdsmetricgstatic.com
n0ewyudnsotlsdsmetricgstatic.com
nbkpwwdnsotlsdsmetricgstatic.com
oawwhadnsotlsdsmetricgstatic.com
qj08nidnsotlsdsmetricgstatic.com
qmhwn3dnsotlsdsmetricgstatic.com
rkpxu1dnsotlsdsmetricgstatic.com
spiipvdnsotlsdsmetricgstatic.com
uhkl8qdnsotlsdsmetricgstatic.com
wr5rednsotlsdsmetricgstatic.com
zj7v8ddnsotlsdsmetricgstatic.com
zx2tb4dnsotlsdsmetricgstati.ccom

___________________________
Paramount Plus
___________________________
cdn.privacy.paramount.com
player-services.paramountplus.com
sparrow.intl.paramountplus.com
www.intl.paramountplus.com
www.paramountplus.com
___________________________
Digicert
___________________________
ocsp.digicert.com

___________________________
Bing
___________________________
bing.com
business.bing.com
edgeservices.bing.com
facvicon-edgeservices.bing.com
gateway.bingviz.microsoftapp.net
speech.platform.bing.com
___________________________
Proton Services
___________________________
drive.api.proton.com
drive.api.proton.me
drive.proton.com
drive.proton.me
mail-api.proton.me
mail.proton.me
proton.com
proton.me
protondrive.com
vpn-api.proton.me
zrh-storage.proton.me

___________________________
Yandex
___________________________
yandex.tu
yandex.ru
yandex.cn

___________________________
DuckDuckGO
___________________________
duckduckgo.com
duckduckgo.ca
duckduckgo.mx
duckduckgo.ru

___________________________
Ask
___________________________
ask.com
ask.ru
ask.mx
ask.org
ask.mx

___________________________
Yelp
___________________________
yelp.ca
yelp.com
yelp.ru

___________________________
Yahoo Countries
___________________________
ca.yahoo.com
cn.yahoo.com
io.yahoo.com
jp.yahoo.com
ka.yahoo.com
kr.yahoo.com
mx.yahoo.com
ny.yahoo.com
ru.yahoo.com
sp.yahoo.com
tk.yahoo.com
uk.yahoo.com
us.yahoo.com

___________________________
Yahoo Domain
___________________________

___________________________
Social Media
___________________________
b-graph.facebook.com
chat-e2ee-mini.facebook.com
edge-mqtt-fallback.facebook.com
edge-mqtt.facebook.com
facvicon-graph.facebook.com
facvicon-web.facebook.com
gateway.facebook.com
graph.facebook.com
web-fallback.facebook.com
web.facebook.com

___________________________
Domain - X - Twitter
___________________________
x.com

___________________________
Discord
___________________________
www.discord.com
seattle4923.discord.media

___________________________
AOL
___________________________
mail.aol.com
api.mail.aol.com
aol.com
_____________________________
Samsung Doamin
_____________________________
acr-ca-prd.samsungcloud.tv
cdn.samsungcloudsolution.com
config.samsungads.com
configprd.samsungcloudsolution.net
gld.push.samsungosp.com
gpm.samsungqbe.com
img-resize-cdn-prod.samsungnyc.com
lcprd1.samsungcloudsolution.net
log-config.samsungacr.com
oempprd.samsungcloudsolution.com
osb-apps-v2.samsungqbe.com
osb-auth-ussvc-v2.samsungqbe.com
osb-tnc-v2-appsacc.samsungqbe.com
osb-ussvc-v2.samsungqbe.com
samsung.com
samsungcloudsolution.net
samsungqbe.com
scpopenapi.samsungcloud.tv
scs.samsungqbe.com
tcp.scs.samsungqbe.com
time.samsungcloudsolution.com
tvpnlinupepgpus.samsungcloud.tv
_xmpp-client._tcp.scs.samsungqbe.com
osb-v1.samsungqbe.com
osb-v2.samsungqbe.com
osb-v3.samsungqbe.com
___________________________
Apple Domain
___________________________
mediaservices.cdn-apple.com

___________________________
Rogers Domain
___________________________
account.rogers.com
facvicon-account.rogers.com
rogers.com
self.events.data.microsoft.com.phub.net.cable.rogers.com
settings.crashlytics.com.phub.net.cable.rogers.com
upload.phub.net.cable.rogers.com
www.rogers.com
___________________________
Banking & Financial
1- JOINHONEY Inc
___________________________
            1

cdn.checkout.joinhoney
cdn.honey.io

d.joinhoney
s.joinhoney
            END
___________________________
Crypto Currentcy
.PHANTOM
___________________________

blocklist.phantom.app
blowfish-blocklist-proxy.phantom.app
data.phantom.app
eppo-proxy.phantom.app
player-services.paramountplus.com
___________________________
VPN(s) Lists Domains
___________________________

___________________________
Cloudflare List Domains
___________________________
d2s6iufwzzeoda.cloudfront.net
d2gk4951n4s5mh.cloudfront.net
d7luqmstmj331.cloudfront.net

___________________________
Amazon Domain
___________________________
api.amazon.com
ocsp.r2m03.amazontrust.com
ns-1053.awsdns-03.org
ns-94.awsdns-11.com
___________________________
Domain
___________________________
linkedin.com
px.ads.linkedin.com
___________________________
Online Shopping Domain
1- Amazon, Inc.
2- Ebay, Inc.
3- 
___________________________
           1
amazon.ca
amazon.ch
amazon.cn
amazon.com
amazon.hk
amazon.in
amazon.jp
amazon.ka
amazon.mx
amazon.net
amazon.ng
amazon.org
amazon.ru
amazon.tk
amazon.uk
          END
          ---
           2
ebay.ca
ebay.cn
ebay.com
ebay.hk
ebay.jp
ebay.mx
ebay.ru
ebay.tk
ebay.uk
          END
          ---
           3

___________________________
News Doamin
___________________________
edition.cnn.com

___________________________
Government Websites
___________________________
australia.gov.au
brasil.gov.br
canada.ca
china.gov.cnjapan.go.jp
europa.eu
france.fr
france.frfrance.fr
germany.de
gov.uk
india.gov.in
mexico.gob.mx
newzealand.govt.nz
russia.gov.ru
saudi.gov.sa
singapore.gov.sg
southafrica.gov.za
southkorea.gov.kr
uae.gov.ae
usa.gov
___________________________
DNS Domains
___________________________
dns.google
dns.msftncsi.com
dns11.quad9.net

___________________________
OPEN-AI and AI CHAT ROOMS
___________________________
api.openai.com
chatgpt.com
openai.com

___________________________
Domain Bitwarden
___________________________
api.bitwarden.com
artifacts.bitwarden.com
bitwarden.com
facvicon-vault.bitwarden.com
vault.bitwarden.com
____________________________
Domain Stripe Services 
____________________________

express.stripe.com
a.stripecdn.com
api.stripe.com
atlas.stripe.com
auth.stripe.com
b.stripecdn.com
billing.stripe.com
buy.stripe.com
c.stripecdn.com
checkout.stripe.com
climate.stripe.com
connect.stripe.com
dashboard.stripe.com
files.stripe.com
hooks.stripe.com
invoice.stripe.com
invoicedata.stripe.com
js.stripe.com
m.stripe.com
m.stripe.network
manage.stripe.com
pay.stripe.com
payments.stripe.com
q.stripe.com
qr.stripe.com
r.stripe.com
stripe.com
terminal.stripe.com
uploads.stripe.com
verify.stripe.com
_________________________
Doamin Fastly service
_________________________

fastly.com
fastly.net
fastlylb.net
fastly.com
fastly.net
fastlylb.net

_________________________
Okta
_________________________

*.kerberos.okta-emea.com
*.kerberos.okta.com
*.kerberos.oktapreview.com
*.mtls.okta-emea.com
*.mtls.okta-gov.com
*.mtls.okta.com
*.mtls.okta.mil
*.mtls.oktapreview.com
*.okta-emea.com
*.okta-gov.com
*.okta.com
*.okta.mil
*.oktacdn.com
*.oktapreview.com

**Comment by CyberCrime-Stoppers** [comment]
You use this for any purpase, towards firewalls, dns filters, or any other info of website in lists
